hnn_core/network_builder.py

```diff
@@ -409,7 +399,7 @@ def _create_cells_and_drives(self, threshold, record_vsoma=False,
                 # this call could belong in init of a _Cell (with threshold)?
                 nrn_netcon = cell.setup_source_netcon(threshold)
                 _PC.cell(cell.gid, nrn_netcon)
-                self.cells.append(cell)
+                self.cells.append(cell)  # XXX fix later
```

Comment:
what is to be fixed here?

Reply:
Right now there are identical lists of cell objects stored in `Network` and `NetworkBuilder`. If we like the cells staying in `Network`, the list in `NetworkBuilder` should be decommissioned.

Reply:
Oh I see. Yeah, I would decommission the `cells` in `NetworkBuilder`. In terms of clean up, `NetworkBuilder` is my next target after a couple more things in `cell`, so this would definitely be up that alley. Can it be done in this PR?

Reply:
Upon reviewing this further, I think it makes sense to leave `NetworkBuilder.cells` as an attribute since it is a clear way for a `NetworkBuilder` instance to keep track of the cells assigned to the current rank (e.g., as exemplified when [destroying Neuron objects](https://github.com/jonescompneurolab/hnn-core/blob/32d33cc96bd788f6c96b97339a98788b2cf3810e/hnn_core/network_builder.py#L543-L560) upon exiting a `NetworkBuilder` context. Note that I have now made this attribute private in https://github.com/jonescompneurolab/hnn-core/pull/321/commits/7b014072c80c859272f00295f17892fea89e0460 since `NetworkBuilder._cells` it is now solely for utility and should never be accessed by even an advanced user.